pages/results.py

```diff
@@ -4,6 +4,7 @@
 from dash import dcc, html, callback, Input, Output
```

Comment:
```suggestion
from dash import dcc, html, callback, dash_table, Input, Output
```
Import convention (should fix error)